src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vedtak/vedtaksperiode/domene/EØSBegrunnelseDB.kt

```diff
@@ -32,4 +32,10 @@ class EØSBegrunnelseDB(
     @Enumerated(EnumType.STRING)
     @Column(name = "begrunnelse", updatable = false)
     val begrunnelse: EØSBegrunnelse,
-)
+) {
+    fun kopier(vedtaksperiodeMedBegrunnelser: VedtaksperiodeMedBegrunnelser): EØSBegrunnelseDB =
+        EØSBegrunnelseDB(
+            vedtaksperiodeMedBegrunnelser = vedtaksperiodeMedBegrunnelser,
+            begrunnelse = this.begrunnelse,
+        )
+}
```

Comment:
Når vi kopierer over fra forrige behandling må vi vel også kopiere over EØS-begrunnelsene? 🤔

Reply:
Vi pleier ikke å dra med begrunnelsene fra forrige behandling i når vi lager en ny behandling. Det som er begrunnet i forrige behandling er allerede kommunisert ut til bruker, så vi ønsker ikke å kommunisere det på nytt.

Reply:
Eller gjøres dette mellom to vedtak i samme behandling? Altså at godkjenner underkjenner det første vedtaket slik at saksbehandler må se på behandlingen på nytt og foreslå et nytt vedtak

Reply:
Ja, dette er i forbindelse med nytt vedtak i samme behandling 👍 Da gir det mening å legge til kopiering av EØS-begrunnelser også tenker jeg!